config/name_node.go

```diff
@@ -8,6 +8,7 @@ import (
 type NameNodeconfig struct {
 	IP                       string //Name node IP
 	InternalRequestsPort     string //The internal requests ports
+	ExternalRequestsPort     string //The external requests ports
```

Comment:
I think we can rename this to just `Port` the same way we did with the data node if I recall correctly.